src/Avalonia.FuncUI/DSL/Flyout.fs

```diff
@@ -0,0 +1,65 @@
+﻿namespace Avalonia.FuncUI.DSL
+
+[<AutoOpen>]
+module Flyout =
+    open Avalonia.Controls
+    open Avalonia.FuncUI.Types
+    open Avalonia.FuncUI.Builder
+    open Avalonia.Controls.Primitives
+    
+    let create (attrs: IAttr<Flyout> list): IView<Flyout> =
+        ViewBuilder.Create<Flyout>(attrs)
+
+    type FlyoutBase with
+            
+        /// <summary>
+        /// A value indicating how the flyout is positioned.
+        /// </summary>
+        static member placement<'t when 't :> FlyoutBase>(value: FlyoutPlacementMode) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<FlyoutPlacementMode>(FlyoutBase.PlacementProperty, value, ValueNone)
+            
+        /// <summary>
+        /// A value indicating flyout show mode.
+        /// </summary>
+        static member showMode<'t when 't :> FlyoutBase>(value: FlyoutShowMode) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<FlyoutShowMode>(FlyoutBase.ShowModeProperty, value, ValueNone)
+            
+    type Flyout with
```

Comment:
I see in the documentation that there's also a `MenuFlyout`, could we also support this in here? I haven't checked in detail, if it'd be too much work we can address it separately but just in case it's just a few more lines.

Reply:
Done, I placed it inside `Flyout.fs` file, not sure again if that corrects, note me if you want it to be in separated file.